src/renderer/widgets/DelegateDetails/ui/YourDelegation.tsx

```diff
@@ -61,10 +61,10 @@ export const YourDelegation = () => {
                   components={{
                     votes: (
                       <AssetBalance
-                        value={activeDelegations[activeAccounts[0]]?.balance
+                        value={activeDelegations[activeAccounts[0]].balance
```

Comment:
Looks unsafe
You need to check if active account and active delegation exists before getting balance